.github/workflows/package.yml

```diff
@@ -0,0 +1,108 @@
+name: Release-Executable
+
+# Pyinstaller requires that executables for each OS are built on that OS
+# This action is intended to build on each of the supported OS's: mac, windows, linux.
+# and then upload all three files to a new release
+
+# reference material:
+# https://data-dive.com/multi-os-deployment-in-cloud-using-pyinstaller-and-github-actions
+# https://anshumanfauzdar.medium.com/using-github-actions-to-bundle-python-application-into-a-single-package-and-automatic-release-834bd42e0670
+
+
+# TODO: actually plan when we want to trigger this job
+#  - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+on:
+  push:
+    tags: release
+    branches: jac/pyinstaller
```

Comment:
Does this break if `jac/pyinstaller` branch gets deleted?

Reply:
No, it's still triggered by any push tagged "release"